src/reflect/scala/reflect/internal/Symbols.scala

```diff
@@ -725,7 +725,7 @@ trait Symbols extends api.Symbols { self: SymbolTable =>
 
     def setFlag(mask: Long): this.type   = { _rawflags |= mask ; this }
     def resetFlag(mask: Long): this.type = { _rawflags &= ~mask ; this }
-    def resetFlags() { rawflags &= TopLevelCreationFlags }
+    def resetFlags() { rawflags = 0 }
```

Comment:
All this flags business really makes me wish we tracked flags in the TypeHistory, just like a symbol's info.